wiki/Tournaments/CFC/2018/en.md

```diff
@@ -0,0 +1,235 @@
+---
+tags:
+- CFC 2018
+- CFC2018
+---
+# Catch French Cup 2018
+
+![](logo.png)
+
+**CFC 2018**, short for **Catch French Cup 2018**, is a one-on-one French-based osu!catch tournament hosted and organized by ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263).
```

Comment:
The **Catch French Cup 2018** (***CFC 2018***) is a one-on-one French-only osu!catch tournament hosted by ![][flag_FR] [Kasumii-sama](https://osu.ppy.sh/users/6177263). It is the 2nd installment of the Catch French Cup 2018.

Reply:
think i should just make it "2nd installment of the catch french cup", instead of adding 2018 at the end

Reply:
yeah my bad